### PR TITLE
Feature/arrow icons

### PR DESCRIPTION
**Pull request by BrockReece**
New atomic arrow components for Croud control.

- New atomic ArrowItem and DrodownIndicatorButton functional components
- Updates the ToggleHeader and Panel components to use new DrodownIndicatorButton
- Fixes the incorrect direction logic in ToggleHeader and fixes corresponding tests.
- Tweaks to generator

Built to spec but to be confirmed by Product

### CODE REVIEW

**Comment by BrockReece** [comment]
Confirmed with product, they are happy with the end results on the Panel and Header components

File: _templates/vue/component/prompt.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = [
     {
         type: 'input',
         name: 'name',
-        message: 'What is the name of your vue component?',
+        message: 'What is the name of your vue component? (PascalCase)',
```
Comment (BrockReece) [inline _templates/vue/component/prompt.js]: @kingy04 is this clear enough or do you think it needs more?

File: styleguide.config.js [hunk 2]
```diff
@@ -19,6 +19,10 @@ module.exports = {
             name: 'Buttons',
             components: 'src/components/shared/buttons/**/*.vue',
         },
+        {
```
Comment (BrockReece) [inline styleguide.config.js]: New directory for icons

File: _templates/vue/functionalComponent/component.ejs.t [hunk 3]
```diff
@@ -19,8 +19,8 @@ to: src/components/shared/<%= dir %>/<%= name %>.vue
         name: '<%= componentName %>',
 
         render(h) {
-            return h('div', "<%= name %>")
-        }
+            return h('div', '<%= name %>')
```
Comment (BrockReece) [inline _templates/vue/functionalComponent/component.ejs.t]: Fix linting issues on functional components

File: _templates/vue/component/prompt.js (hunk 1 above)
Comment (kingy04) [reply _templates/vue/component/prompt.js]: This is fine thanks

**Pull request merged**